scripts/build.sh

```diff
@@ -62,27 +62,36 @@ mingw)
     make -j
     CURL="$(download "$MINGW_CURL_WIN64_BLOB")"
     cp "$CURL"/curl-*-mingw/bin/libcurl-x64.dll output/libcurl.dll
-    zip -r -j "$WORKSPACE/lpac-windows-x86_64-mingw.zip" output/*
+    [ -f "$(which zip)" ] && zip -r -j "$WORKSPACE/lpac-windows-x86_64-mingw.zip" output/*
     ;;
 woa-mingw)
     TOOLCHAIN="$(download "$MINGW32_TOOLCHAIN_BLOB")"
     cmake "$WORKSPACE" -DCMAKE_TOOLCHAIN_FILE=./cmake/linux-mingw64-woa.cmake "-DTOOLCHAIN_BIN_PATH=$TOOLCHAIN/bin"
     make -j
     CURL="$(download "$MINGW_CURL_WIN64A_BLOB")"
     cp "$CURL"/curl-*-mingw/bin/libcurl-arm64.dll output/libcurl.dll
-    zip -r -j "$WORKSPACE/lpac-windows-arm64-mingw.zip" output/*
+    [ -f "$(which zip)" ] && zip -r -j "$WORKSPACE/lpac-windows-arm64-mingw.zip" output/*
     ;;
 woa-zig)
     cmake "$WORKSPACE" -DCMAKE_TOOLCHAIN_FILE=./cmake/aarch64-windows-zig.cmake
     make -j
     CURL="$(download "$MINGW_CURL_WIN64A_BLOB")"
     cp "$CURL"/curl-*-mingw/bin/libcurl-arm64.dll output/libcurl.dll
-    zip -r -j "$WORKSPACE/lpac-windows-arm64-zig.zip" output/*
+    [ -f "$(which zip)" ] && zip -r -j "$WORKSPACE/lpac-windows-arm64-zig.zip" output/*
     ;;
 *)
     echo "Usage: $0 {make,debian,mingw,woa-mingw,woa-zig}"
     exit 1
     ;;
 esac
 
+rm -rf "$WORKSPACE/build"
+mkdir "$WORKSPACE/build"
+if [ -d "$BUILD/output" ]; then
+    cp -r "$BUILD/output" "$WORKSPACE/build"
+fi
+if [ -n "$(find "$BUILD" -name 'lpac-*.deb')" ]; then
```

Comment:
nice to know that a .deb package can be created, but how do I even create that? I don't see it listed anywhere in the installation instructions.